chatbot/services/response_matcher.py

```
@@ -0,0 +1,241 @@
+"""
+سرویس تطبیق پاسخ‌ها
+Response Matcher Service
+"""
+
+import re
+from typing import List, Optional, Dict, Any
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)